### PR TITLE
RPC DQM Active Channel Fraction

### PR DESCRIPTION
**Pull request by acimmino**
Added 2 summary histograms to monitor active channel fraction in DQM.

### CODE REVIEW

**Comment by deguio** [comment]
please test

**Comment by deguio** [comment]
@acimmino 
ok, I didn't see it..  :)

F.

**Comment by deguio** [comment]
+1

File: DQM/RPCMonitorClient/interface/RPCOccupancyTest.h [hunk 1]
```diff
@@ -36,11 +34,16 @@ class RPCOccupancyTest:public RPCClient {
   std::vector<MonitorElement *>  myOccupancyMe_;
   std::vector<RPCDetId>   myDetIds_;
   int prescaleFactor_;
- 
+
+  float totalActive_, totalStrips_;
```
Comment (davidlange6) [inline DQM/RPCMonitorClient/interface/RPCOccupancyTest.h]: @acimmino - why floats? these are being used as counters, so an unsigned int would be the usual choice.

**Comment by acimmino** [comment]
Hi David,

Your objection to the use of "floats" instead of "ints" is correct. I made the fix and pushed the changes.

Should I do the same for the 80X (PR #14258)?

Cheers,
Anna


________________________________

**Comment by dmitrijus** [comment]
This PR got somewhat lost :(
Please rebase if you still want it to be merged.

**Pull request closed without merging**